pkg/utils/k8sutil.go

```diff
@@ -453,21 +453,33 @@ func CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic,
 		Do().Into(&f)
 	if err != nil {
 		if k8sErrors.IsNotFound(err) {
+			funcLabels := map[string]string{}
+			for _, label := range labels {
+				k, v := getKV(label)
```

Comment:
Do you really want to use `getKV` here at all?  It seems odd to me that you support substituting environment values for `--labels` but _not_ for `--env`.  Tbh, I'd drop the feature (remove `os.Getenv` calls from `getKV`) until there's actually a use case that makes it worthwhile to support - it's not like it's hard to do `--label=foo=$foo` in the shell calling `kubeless`.